packages/components/table/src/table.tsx

```diff
@@ -34,13 +36,13 @@ export interface TableProps
  */
 export const Table = forwardRef<TableProps, "table">((props, ref) => {
   const styles = useMultiStyleConfig("Table", props)
-  const { className, ...tableProps } = omitThemingProps(props)
+  const { className, layout = "auto", ...tableProps } = omitThemingProps(props)
```

Comment:
We don't need a default value

```suggestion
  const { className, layout, ...tableProps } = omitThemingProps(props)
```